src/terminal/input/terminalInput.cpp

```diff
@@ -485,6 +485,17 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
         keyEvent.DeactivateModifierKey(ModifierKeyState::RightAlt);
     }
 
+    const auto senderFunc = [this](const std::wstring_view seq) noexcept
+    {
+        _SendInputSequence(seq);
+    };
+
+    // If a modifier key was pressed, then we need to try and send the modified sequence.
+    if (keyEvent.IsModifierPressed() && _searchWithModifier(keyEvent, senderFunc))
+    {
+        return true;
+    }

```

Comment:
Yea moving this seems fine to me, it's not like the code that _was_ before it was having any effect on this block, so it can life here just fine.